docs/containers.conf.5.md

```diff
@@ -256,6 +256,9 @@ conmon_path=[
 ]
 ```
 
+**container_engine_env**=[]
+Additional environment variables to apply to the container engine.  For example "http_proxy=internal.proxy.company.com"
```

Comment:
How about something like:

"Environment variables to be used when running the container engine. Example....
Note these environment variables will not be used within the container.  Set the env section under containers, if you want
to set environment variables for the container.
```